lib/js/components/Cards/Card/index.ts

```diff
@@ -0,0 +1,3 @@
+import Card from './Card.vue';
```

Comment:
I don't understand why we add these index files if we don't use them. Do you plan to use them in `lib/js/index.ts` or should they be removed?

Reply:
After discussion: All imports with `.vue` in `vue` files are replaced by import folder which contains `index.ts` file.